pkg/controllers/provisioning/provisioner.go

```diff
@@ -161,3 +190,91 @@ func (p *Provisioner) FilterProvisionable(ctx context.Context, pods []*v1.Pod) [
 	}
 	return provisionable
 }
+
+func (p *Provisioner) launch(ctx context.Context, constraints *v1alpha5.Constraints, packing *binpacking.Packing) error {
```

Comment:
Why did we collapse the launcher into this file btw? Don't feel too strongly about it either way

Reply:
It will become more apparent as I continue to refactor this, but essentially the logic got small enough that it didn't really make sense to have all of the overhead ~70 lines of code. Additionally, I'm going to be moving other provisioner behavior (e.g. batching/scheduling) elsewhere. You can think about the provisioner as becoming a renamed launcher.